server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts

```diff
@@ -87,6 +87,38 @@ clinicalDetailsRoute.post('/backgroundDiseases', (request: Request, response: Re
     });
 });
 
+clinicalDetailsRoute.get('/isolationSources', (request: Request, response: Response) => {
+    logger.info({
+        service: Service.SERVER,
+        severity: Severity.LOW,
+        workflow: 'Fetching Isolation Source',
+        step: 'start DB request',
+        investigation: response.locals.epidemiologynumber,
+        user: response.locals.user.id
+    })
+    graphqlRequest(GET_ISOLATION_SOURCES, response.locals).then((result: any) => {
+        logger.info({
+            service: Service.SERVER,
+            severity: Severity.LOW,
+            workflow: 'Fetching Isolation Source',
+            step: 'got response from DB',
+            investigation: response.locals.epidemiologynumber,
+            user: response.locals.user.id
+        })
+        response.send(result.data)
```

Comment:
parse the data, dont send the response, send only the array

Reply:
Fixed